docs/retiring-and-removing-documents.md

```diff
@@ -0,0 +1,61 @@
+# Retiring and removing documents
+
+We need to allow users to unpublish content from GOV.UK. In Content Publisher we are replacing
+unpublish terminology with "retired" and "removed".
+
+Retired content is still displayed on GOV.UK, but there is an explanation box on the page that describes
+why the content is out of date.
+
+Removed content returns a 410 gone page to the user. If an explanatory note or a alternative path have been provided,
+they will be displayed in the body of the page.
+
+Removed and redirected content redirects users to another page on GOV.UK
+
+Environment variables are being used to pass parameters to the rake tasks.
+
+## Retiring documents
+
+Required parameters:
+
+- CONTENT_ID
+- NOTE
+
+Optional parameters:
+
+- LOCALE (set to "en" by default)
+
+```
+rake unpublish:retire_document CONTENT_ID='a-content-id' NOTE='A note'
```

Comment:
Just a suggestion and not a blocker but since all these commands have content_id as an argument shall we pass that in through the normal rake way e.g. `rake unpublish:retire_document[a-content-id] NOTE='A note'` ? Just feels a bit more succinct and it would save you needing lines like https://github.com/alphagov/content-publisher/pull/510/files#diff-b0f8989c20523cc86786623565a5f99cR21 and associated tests

Reply:
Yeah, that's a good point. I could change them all to only use ENV VARS for the optional arguments.

Reply:
I think you'd still probably want an env var for a note even when that's required. But that's mostly because of rake's yucky syntax and the comma pains associated.

Reply:
True, and to keep it consistent with the other tasks that have note as an optional parameter.